201-traffic-manager-vm-zones/azuredeploy.json

```diff
@@ -64,11 +66,12 @@
     "publicIPAddressName": "myPublicIP",
     "vmName": "MyUbuntuVM",
     "virtualNetworkName": "MyVNET",
+    "networkSecurityGroupName": "MyNSG",
     "subnetRef": "[resourceId('Microsoft.Network/virtualNetworks/subnets/', variables('virtualNetworkName'), variables('subnetName'))]",
     "linuxImage": {
       "publisher": "Canonical",
       "offer": "UbuntuServer",
-      "sku": "16.04.0-LTS",
+      "sku": "19.10-DAILY",
```

Comment:
I think we should either move this to a parameter or grab an LTS version...  otherwise I suspect we'll remove the image and this will start failing...

Reply:
VM SKU moved to parameters

Reply:
if you can get it to work with 19.04 (not daily) then let's just hard code that version